documentation/source/refcard.rst

```diff
@@ -0,0 +1,93 @@
++------------------------+-----------------------------------------------------------------+
+| Assign immediately     | ``dut.mysignal.setimmediatevalue(0xFF00)``                      |
++------------------------+-----------------------------------------------------------------+
+| Assign metavalue       | ``dut.mysignal <= BinaryValue("X")``                            |
++------------------------+-----------------------------------------------------------------+
+| Read                   | | ``val = dut.mysignal.value``                                  |
+|                        | | (``mysig = dut.mysignal`` *creates an alias/reference)*       |
++------------------------+-----------------------------------------------------------------+
+| Bit slice              | | ``mybit = dut.myarray[0].value``                              |
+|                        | | ``mybits = dut.mysignal.value[0]``                            |
++------------------------+-----------------------------------------------------------------+
+| Convert                | | ``val = dut.mysignal.value.integer``                          |
+|                        | | ``val = dut.mysignal.value.binstr``                           |
++------------------------+-----------------------------------------------------------------+
+| Check                  | ``assert dut.mysignal.value == exp, "Not as expected!"``        |
++------------------------+-----------------------------------------------------------------+
+| Logging                | ``dut._log.info("Value is", dut.mysignal.value)``               |
++------------------------+-----------------------------------------------------------------+
+|                                                                                          |
++------------------------+-----------------------------------------------------------------+
+| Wait time              | ``await cocotb.triggers.Timer(12, "ns")``                       |
++------------------------+-----------------------------------------------------------------+
+| Generate clock         | ``clk = cocotb.fork(Clock(dut.clk, 12, "ns").start())``         |
++------------------------+-----------------------------------------------------------------+
+| Wait for signal edge   | | ``await cocotb.triggers.RisingEdge(dut.mysignal)``            |
+|                        | | ``await cocotb.triggers.FallingEdge(dut.mysignal)``           |
+|                        | | ``await cocotb.triggers.Edge(dut.mysignal)``                  |
++------------------------+-----------------------------------------------------------------+
+| Run coros concurrently | | ``task_0 = cocotb.fork(coro_0())``                            |
+|                        | | ``task_1 = cocotb.fork(coro_1())``                            |
+|                        | | ``task_2 = cocotb.scheduler.start_soon(coro)``                |
```

Comment:
I might not repeat the use of `fork`, but use a line to describe the difference between `start_soon` and `fork`?

Reply:
I removed the 2nd for line but didn't add an explanation - we aren't explaining things anywhere else here (except for pitfalls); after all this is only meant to be a reminder of what you already know.